README.md

```diff
@@ -1,5 +1,75 @@
 daml-dit-arcade-index
 ====
 
-List of repositories included in the
-[project:DABL](https://projectdabl.com/) Arcade.
+This repository defines a list of packages that are included in the [Daml Hub](https://projectdabl.com/) Arcade. It consists of both **Sample Apps** and **Integrations**, whose locations are declared in the `repositories.yaml` index within this project.
+
+# Adding a Sample App
+
+Do you have a fully-fledged Daml application that runs on Daml Hub with a DAR model, triggers, custom UI, and other artifacts? Want to see it included as a one-click deployable Sample App? Read on.
+
+All Sample Apps (and Integrations) are consumed by Daml Hub in the form of [DIT files](https://github.com/digital-asset/daml-dit-api). In short, a DIT file is a single package that contains the metadata and deployable entities that make up your application.
+
+There are 5 straightforward tasks to getting your application ready and published on Daml Hub's arcade:
+
+1. [Configure](#Configure)
+2. [Build](#Build)
+3. [Release](#Release)
+4. [List](#List)
+5. [Update](#Update)
+
+> _Note_: Digital Asset reserves the right to make the ultimate determination of officially including your application in the arcade. However, an application DIT file can still be manually uploaded by any user to a Daml Hub ledger -- outside of the arcade -- so it may still prove a useful way of packaging, testing, and/or distributing non-official Daml Hub apps, depending on your use case.
```

Comment:
May be worthwhile to explicitly either describe the process for doing this or link to the DAML:Hub docs..